### ISSUE TITLE
"Ignore" check box (in INSERT tab) does not uncheck when all the fields are edited through popup windows

### ISSUE THREAD

**Issue by diyoyo** [phpmyadmin/phpmyadmin#14502]
### Steps to reproduce
1.  Create a table with let's say 2 columns with foreign key
2. In the related tables (those referred to by the keys), add many entries, so that it reaches the threshold of PhpMyAdmin where the < select > becomes a popup, for displaying the options.

### Expected behaviour
The "Ignore" checkbox should become unchecked for that entry, since fields are filled.

### Actual behaviour
The js listener does not understand that fields have been filled, and thus, the "Ignore" button is not set to the unchecked state!

### Server configuration
**phpMyAdmin version:** 4.8.3, 5.0.0-dev

**Comment by williamdes** [comment]
Demo servers (root, no password)
- ~~[`5.0.0-dev`](https://demo.phpmyadmin.net/master-config/)~~
- [`4.8.x`](https://demo.phpmyadmin.net/QA_4_8/)
Reproduced on demo server.
5.0.0-dev has bug that prevents reproducing this issue.


### Test data
```sql
SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";

CREATE TABLE a (
  a varchar(3) NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=latin1;

INSERT INTO a (a) VALUES
('\"r\"'),
('&1');

CREATE TABLE b (
  aa varchar(3) NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=latin1;

INSERT INTO b (aa) VALUES
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('\"r\"'),
('&1'),
('&1'),
('&1'),
('&1'),
('&1'),
('\'-y'),
('\'-y'),
('\'-y'),
('\'-y'),
('\'-y'),
('(\'-'),
('(\'-'),
('(\'-'),
('(\'-'),
('(\'-'),
('(uq'),
('(uq'),
('(uq'),
('(uq'),
('(uq'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('-è-'),
('1'),
('1'),
('1'),
('1'),
('1'),
('2'),
('2'),
('2'),
('2'),
('2'),
('2'),
('2'),
('2'),
('2'),
('2'),
('22'),
('22'),
('22'),
('22'),
('22'),
('222'),
('222'),
('222'),
('222'),
('222'),
('2é\"'),
('2é\"'),
('2é\"'),
('2é\"'),
('2é\"'),
('3'),
('3'),
('3'),
('3'),
('3'),
('33'),
('33'),
('33'),
('33'),
('33'),
('333'),
('333'),
('333'),
('333'),
('333'),
('333'),
('333'),
('333'),
('333'),
('333'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('528'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('85H'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('889'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('AZD'),
('ddd'),
('ddd'),
('ddd'),
('ddd'),
('ddd'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DOL'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('DUJ'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('EDE'),
('eee'),
('eee'),
('eee'),
('eee'),
('eee'),
('ERG'),
('ERG'),
('ERG'),
('erg'),
('ERG'),
('ERG'),
('ERG'),
('erg'),
('ERG'),
('ERG'),
('ERG'),
('erg'),
('ERG'),
('ERG'),
('ERG'),
('erg'),
('ERG'),
('ERG'),
('ERG'),
('erg'),
('ezf'),
('ezf'),
('ezf'),
('ezf'),
('ezf'),
('fbr'),
('fbr'),
('fbr'),
('fbr'),
('fbr'),
('fef'),
('FEF'),
('FEF'),
('FEF'),
('fef'),
('FEF'),
('FEF'),
('FEF'),
('fef'),
('FEF'),
('FEF'),
('FEF'),
('fef'),
('FEF'),
('FEF'),
('FEF'),
('fef'),
('FEF'),
('FEF'),
('FEF'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FET'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRE'),
('FRF'),
('FRF'),
('FRF'),
('frf'),
('FRF'),
('FRF'),
('FRF'),
('frf'),
('FRF'),
('FRF'),
('FRF'),
('frf'),
('FRF'),
('FRF'),
('FRF'),
('frf'),
('FRF'),
('FRF'),
('FRF'),
('frf'),
('ilu'),
('ilu'),
('ilu'),
('ilu'),
('ilu'),
('iul'),
('iul'),
('iul'),
('iul'),
('iul'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfe'),
('rfr'),
('rfr'),
('rfr'),
('rfr'),
('rfr'),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('RG('),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rg5'),
('rgu'),
('rgu'),
('rgu'),
('rgu'),
('rgu'),
('rh\''),
('rh\''),
('rh\''),
('rh\''),
('rh\''),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('rve'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('SXS'),
('uil'),
('uil'),
('uil'),
('uil'),
('uil'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('Y(-'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zdd'),
('zr'),
('zr'),
('zr'),
('zr'),
('zr');

CREATE TABLE `issue-14274` (
  id int(11) NOT NULL,
  z geometry NOT NULL,
  y point DEFAULT NULL,
  x linestring DEFAULT NULL,
  w multipolygon DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

;


ALTER TABLE a
  ADD KEY a (a);

ALTER TABLE b
  ADD KEY aa (aa);

ALTER TABLE issue-14274
  ADD PRIMARY KEY (id);


ALTER TABLE issue-14274
  MODIFY id int(11) NOT NULL AUTO_INCREMENT;


ALTER TABLE a
  ADD CONSTRAINT a_ibfk_1 FOREIGN KEY (a) REFERENCES `b` (aa);
```

**Comment by williamdes** [comment]
cc @anny21

This is a good one and occurs also with the GIS edit modal

**Comment by anny21** [comment]
Thank you @williamdes.. i'm looking into it, I will send you a message on twitter soon. Thank you

**Comment by anny21** [comment]
@williamdes I sent you a message on twitter 😭

**Comment by williamdes** [comment]
Thank you @anny21 for your fix #17082 !
It works perfectly !
